### PR TITLE
Adding environment variables from facts

### LINKED ISSUE #63: EC2 tags to find their way to shell environment variables
This would make life really easy e.g. in a Jenkins pipeline. Let's say the pattern is something like `$EC2_TAG_Env` for a tag named `Env`. @alvagante @shapi78 What do you think is the best way to make this happen?

### PR DESCRIPTION
**Pull request by shapi78**
Pass puppet facts to environment varibles to use on Jenkins
Fixes #63

### CODE REVIEW

**Review by alvagante — COMMENTED** [review]
Looks ok.

File: Puppetfile [hunk 1]
```diff
@@ -29,7 +29,8 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts'
+  :git => 'https://github.com/graduway/ec2tagfacts',
+  :branch => 'prod-env'
```
Comment (antgel) [inline Puppetfile]: Please open a PR to merge this into our master.

File: Puppetfile [hunk 2]
```diff
@@ -65,7 +66,7 @@ mod 'puppetlabs/dummy_service', :latest
 # mod 'puppet-openvpn', '7.1.0'
 mod 'openvpn',
   :git => 'https://github.com/graduway/puppet-openvpn.git',
-  :branch => 'mail'
+  :branch => 'mail-env'
```
Comment (antgel) [inline Puppetfile:69]: Please open a PR to merge this into our master.

**Review by antgel — CHANGES_REQUESTED** [review]
Code great, don't want production code sitting on branches, please open PRs to `master` and edit these commits to pull from `master`.

**Review by antgel — APPROVED** [review]

File: Puppetfile [hunk 3]
```diff
@@ -97,3 +98,4 @@ mod 'puppetlabs-iis', :latest
 mod 'puppetlabs/accounts', :latest
 mod 'oxc-postfix', '1.0.0'
 ## mod 'puppet-fail2ban', '2.2.0'
+mod 'example42-profile', '2.0.4'
```
Comment (alvagante) [inline Puppetfile:101]: Note that current upstream psick module has the psick::profile class which does the same

File: site/profile/manifests/profile.pp [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class profile::profile (
+  	String $template  = '',
```
Comment (alvagante) [inline site/profile/manifests/profile.pp:2]: Here the $template variable does nothing, can be removed

**Pull request merged**